web/src/user_group_edit.ts

```diff
@@ -1632,7 +1640,7 @@ function setup_dropdown_filters_widget(): void {
         widget_name: "user_group_visibility_settings",
         item_click_callback: filter_click_handler,
         $events_container: $("#more_options_user_group"),
-        default_id: initial_visible_groups,
+        default_id: get_relevant_filter(),
```

Comment:
I don't see any need of adding this function. Insted `initial_visible_user_groups` can be used here, and then `update_displayed_groups` can just compute the selected filter value dropdown.